### PR TITLE
delete book image

### PR DESCRIPTION
**Pull request by ghassanzadeh**
# Summary
- Added a fab for deleting a book's photo on Library Edit Book
- The owner can remove the current photo of their books

# Screenshots
with_photo_dialog
without_photo

### CODE REVIEW

File: app/src/main/java/com/example/unlibrary/library/LibraryRepository.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 package com.example.unlibrary.library;
 
+import android.net.Uri;
```
Comment (cyrusdiego) [inline app/src/main/java/com/example/unlibrary/library/LibraryRepository.java:11]: i think this was unintended?

File: app/src/main/res/layout/fragment_library_edit_book.xml [hunk 2]
```diff
@@ -27,6 +27,20 @@
             tools:imagePath="@{viewModel.currentBook.photo}"
             tools:imageUri="@{viewModel.takenPhoto}" />
 
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
```
Comment (cyrusdiego) [inline app/src/main/res/layout/fragment_library_edit_book.xml:30]: Should we set the visibility of this button if there's no photo?

**Review by cyrusdiego — COMMENTED** [review]
Left some comments, overall looks good :D

**Review by armiantos — COMMENTED** [review]
Cyrus sums it up. Concise code for a complete user story. Good work!

**Review by armiantos — APPROVED** [review]
LGTM :beer:

**Review by calebschoepp — APPROVED** [review]
LGTM!

**Review by cyrusdiego — APPROVED** [review]

**Pull request merged**